openapi/openapi.json

```diff
@@ -992,91 +1326,116 @@
-          "required" : false,
-          "type" : "array",
-          "items" : {
-            "type" : "string"
-          },
-          "collectionFormat" : "multi"
+          "style" : "form",
+          "schema" : {
+            "type" : "array",
+            "items" : {
+              "type" : "string"
+            }
+          }
         }, {
           "name" : "q",
           "in" : "query",
           "description" : "A full text search. Optionally, it's possible to search on a field using this syntax: {fieldname}:{text}",
-          "required" : false,
-          "type" : "array",
-          "items" : {
-            "type" : "string"
-          },
-          "collectionFormat" : "multi"
-        }, {
-          "name" : "dateformat",
-          "in" : "query",
+          "style" : "form",
+          "schema" : {
+            "type" : "array",
+            "items" : {
+              "type" : "string"
+            }
+          }
+        }, {
+          "name" : "dateformat",
+          "in" : "query",
           "description" : "The format of dates. This parameter should be set only if a date field is queried in `f` param; when using `gt`, `lt`, `gte`, `lte` and `range` operations",
-          "required" : false,
-          "type" : "string"
+          "schema" : {
+            "type" : "string"
+          }
         }, {
           "name" : "righthand",
           "in" : "query",
           "description" : "If righthand = true, the passed WKT should be counter clock-wise; otherwise, ARLAS-server will attempt to parse it as the \"Complementary\" Polygon on the other facet of the planet. \n Inversely, If righthand = false, the passed WKT should be clock-wise; otherwise, ARLAS-server will attempt to parse it as the \"Complementary\" Polygon on the other facet of the planet. ",
-          "required" : false,
-          "type" : "boolean",
-          "default" : true
+          "schema" : {
+            "type" : "boolean",
+            "default" : true
+          }
         }, {
           "name" : "pretty",
           "in" : "query",
           "description" : "Pretty print",
-          "required" : false,
-          "type" : "boolean",
-          "default" : false
+          "schema" : {
+            "type" : "boolean",
+            "default" : false
+          }
         }, {
           "name" : "max-age-cache",
           "in" : "query",
           "description" : "max-age-cache",
-          "required" : false,
-          "type" : "integer",
-          "format" : "int32"
+          "schema" : {
+            "type" : "integer",
+            "format" : "int32"
+          }
         } ],
         "responses" : {
           "200" : {
             "description" : "Successful operation",
-            "schema" : {
-              "$ref" : "#/definitions/ComputationResponse"
+            "content" : {
+              "application/json;charset=utf-8" : {
+                "schema" : {
+                  "type" : "array",
```

Comment:
will be fixed by removing the @array annotation

Reply:
ok